### PR TITLE
New consts & removal of flex

### PR DESCRIPTION
**Pull request by tajo**
- new color used by navigation
- new special media query used by navigation
- removal of flex from the form component, it's not needed and it breaks cf-styleguide

### CODE REVIEW

File: packages/cf-style-provider/src/createRenderer.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ const mediaQueries = {
   mobileWide: `@media (min-width: ${variables.breakpoints.mobileWide})`,
   tablet: `@media (min-width: ${variables.breakpoints.tablet})`,
   desktop: `@media (min-width: ${variables.breakpoints.desktop})`,
-  desktopLarge: `@media (min-width: ${variables.breakpoints.desktopLarge})`
+  desktopLarge: `@media (min-width: ${variables.breakpoints.desktopLarge})`,
+  hover: `@media (hover)`
```
Comment (koddsson) [inline packages/cf-style-provider/src/createRenderer.js]: Are we relying on this somewhere or is this just a enhancement? It doesn't seem to [be supported in Firefox](https://caniuse.com/#feat=css-media-interaction).

File: packages/cf-style-provider/src/createRenderer.js [hunk 2]
```diff
@@ -47,6 +48,7 @@ const createRenderer = opts => {
     enhancers,
     selectorPrefix: usedOpts.selectorPrefix,
     mediaQueryOrder: [
```
Comment (koddsson) [inline packages/cf-style-provider/src/createRenderer.js]: This list seems weird and redundant. Can't this just be `Object.values(mediaQueries).map(removePrefix)`?

File: packages/cf-style-provider/src/createRenderer.js (hunk 1 above)
Comment (tajo) [inline packages/cf-style-provider/src/createRenderer.js]: Yep, I use it strictly for enhancements.

File: packages/cf-style-provider/src/createRenderer.js (hunk 2 above)
Comment (tajo) [inline packages/cf-style-provider/src/createRenderer.js]: It theory. But objects don't have to preserve the order of properties so it's not safe. That's why that API is using an array in the first place.

**Pull request merged**